### ISSUE TITLE
Make build work on Linux

### ISSUE THREAD

**Issue by Kami** [StackStorm-Exchange/web#19]
Right now npm install fails (fsevent is not available for Linux), which I assume causes partial installation and subsequent build failures.

It would still be good to have this even when #18 is in place (should always make it possible to build and replicate things locally).

```bash
npm ERR! Linux 4.8.13-300.fc25.x86_64
npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "install"
npm ERR! node v6.9.1
npm ERR! npm  v3.10.8
npm ERR! code EBADPLATFORM

npm ERR! notsup Unsupported platform for fsevents@1.0.15: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm ERR! notsup Valid OS:    darwin
npm ERR! notsup Valid Arch:  any
npm ERR! notsup Actual OS:   linux
npm ERR! notsup Actual Arch: x64

npm ERR! Please include the following file with any support request:
npm ERR!     /home/kami/w/stackstorm-exchange/web/npm-debug.log
```

**Comment by Kami** [comment]
This might be related https://github.com/npm/npm/issues/14042

**Comment by Kami** [comment]
Fixed in #20 (removed optional fsevents dependency).
